.github/dependabot.yml

```diff
@@ -8,7 +8,24 @@ updates:
           time: '12:00'
           timezone: 'America/Los_Angeles'
       open-pull-requests-limit: 5
+      ignore:
+          - dependency-name: '@spectrum-css/*'
       reviewers:
           - 'hunterloftis'
           - 'najikahalsema'
           - 'westbrook'
+
+    # This sets up a daily check for Spectrum CSS updates
+    # focused on the @spectrum-css/* packages
+    - package-ecosystem: 'npm'
+      directory: '/'
+      schedule:
+          interval: 'daily'
```

Comment:
Should this has triggered a PR based on the CSS release that went out last week? Like https://www.npmjs.com/package/@spectrum-css/picker/v/7.0.17. If so, what happened? If not, what _should_ it have done or _will_ it do in the future?

Let's clarify that this configuration is/works as expected before reviewing #3798 and #3799.

Reply:
No it should not have because as mentioned in this PR - https://github.com/adobe/spectrum-web-components/pull/3799#issuecomment-1814267518 - duplicate package-ecosystem entries are invalid in the schema at the moment.